tests/e2e/test_rosacli_cluster.go

```diff
@@ -3510,3 +3510,117 @@ var _ = Describe("Sts cluster creation supplemental testing",
 				Expect(zones).To(Equal("[us-east-2a us-east-2b us-east-2c]"))
 			})
 	})
+
+var _ = Describe("Sts cluster with BYO oidc flow creation supplemental testing",
+	labels.Feature.Cluster,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			rosaClient         *rosacli.Client
+			clusterService     rosacli.ClusterService
+			customProfile      *profilehandler.Profile
+			clusterID          string
+			ocmResourceService rosacli.OCMResourceService
+			testingClusterName string
+		)
+		BeforeEach(func() {
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			ocmResourceService = rosaClient.OCMResource
+
+			By("Prepare custom profile")
```

Comment:
Is there any rule of thumb on when we choose to create a custom profile like here and when we make a full profile in the yaml file?

Reply:
A full profile would be created for day-1 cluster creation ,also using for day-2 test case, for each profile should have one CI job using it.
For the test case labeled "labels.Runtime.Day1Supplemental", which will be executed by the supplemental CI job, means no prepared cluster for this kind of cases, it needs to create a cluster on demand, using custom profile is one way to prepare the cluster for it.

Reply:
By the way, the cases labeled "labels.Runtime.Day1Supplemental" itself should have the full cycle of the created cluster. While the cases with the label like day-2, don't need the cleanup step which will be done by the job step.

Reply:
Interesting, thank you for the explanation!